test/test_jit_cuda_fuser.py

```diff
@@ -1631,30 +1631,50 @@ def t_wb(shapes: List[int], x, w, b, eps: float, cudnn: bool):
     @unittest.skipIf(GRAPH_EXECUTOR != ProfilingMode.PROFILING,
                      "Requires fusion optimization pass to be effective")
     def test_norm_half_layer(self):
+        op = "aten::batch_norm"
```

Comment:
curious if it makes sense to just put these un-skips in setUp()?

Reply:
That's actually a good idea. Would be easier to revert later. Will do